docs/guides/getting-started/prerequisites.md

```diff
@@ -47,6 +47,24 @@ Alternatively, you could use `winget` to install rustup using the following comm
 winget install --id Rustlang.Rustup
 ```
 
+#### 3. MSVC toolchain as default
+
+In order to be able to install tools like `tauri-cli` and `trunk` you need to make sure that the proper toolchain is set as default.
+
+To display the current configured default toolchain please run the following command:
+
+```powershell
+rustup default
+```
+
+The command should return either `i686-pc-windows-msvc`, `x86_64-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.
+
+If the command returns a different toolchain, please configure the default toolchain in the following way: 
+
+```powershell
+rustup default stable-msvc
+```
+
```

Comment:
To make it a bit more clear what direction i thought we should go here:
`````suggestion
:::caution MSVC toolchain as default

For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.

If you already had Rust installed, you can make sure the correct toolchain is installed by running this command:

```powershell
rustup default stable-msvc
```

:::
`````

This way we could catch both cases:
- the installer selected gnu for whatever reason (user had mingw installed but not msvc or something)
- the user already had rust installed, but with the wrong toolchain

P.S. no need to accept my suggestion as-is, just wanted to visualize it.